[2024-08-12T06:00:59.280+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_monthly_data.predict_and_save manual__2024-08-12T06:00:54.283207+00:00 [queued]>
[2024-08-12T06:00:59.285+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_monthly_data.predict_and_save manual__2024-08-12T06:00:54.283207+00:00 [queued]>
[2024-08-12T06:00:59.285+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T06:00:59.291+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): predict_and_save> on 2024-08-12 06:00:54.283207+00:00
[2024-08-12T06:00:59.298+0000] {standard_task_runner.py:57} INFO - Started process 7285 to run task
[2024-08-12T06:00:59.302+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'generate_monthly_data', 'predict_and_save', 'manual__2024-08-12T06:00:54.283207+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/monthly_prediction.py', '--cfg-path', '/tmp/tmpeso7nl_a']
[2024-08-12T06:00:59.304+0000] {standard_task_runner.py:85} INFO - Job 104: Subtask predict_and_save
[2024-08-12T06:00:59.360+0000] {task_command.py:415} INFO - Running <TaskInstance: generate_monthly_data.predict_and_save manual__2024-08-12T06:00:54.283207+00:00 [running]> on host fa284eed0508
[2024-08-12T06:00:59.487+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='generate_monthly_data' AIRFLOW_CTX_TASK_ID='predict_and_save' AIRFLOW_CTX_EXECUTION_DATE='2024-08-12T06:00:54.283207+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-12T06:00:54.283207+00:00'
[2024-08-12T06:00:59.527+0000] {logging_mixin.py:151} INFO - gs://mlflow-artifacts-bt/1/fab1118790cd421ab4d4d45e0b254b51/artifacts/models
[2024-08-12T06:01:00.007+0000] {logging_mixin.py:151} WARNING - Downloading artifacts:   0%|          | 0/9 [00:00<?, ?it/s]
[2024-08-12T06:01:00.007+0000] {logging_mixin.py:151} WARNING - Downloading artifacts:  11%|#1        | 1/9 [00:00<00:00, 16980.99it/s]
[2024-08-12T06:01:00.007+0000] {logging_mixin.py:151} WARNING - Downloading artifacts:  22%|##2       | 2/9 [00:00<00:00, 6502.80it/s] 
[2024-08-12T06:01:00.058+0000] {logging_mixin.py:151} WARNING - Downloading artifacts:  33%|###3      | 3/9 [00:00<00:00, 58.89it/s]  
[2024-08-12T06:01:00.069+0000] {logging_mixin.py:151} WARNING - Downloading artifacts:  44%|####4     | 4/9 [00:00<00:00, 64.90it/s]
[2024-08-12T06:01:00.080+0000] {logging_mixin.py:151} WARNING - Downloading artifacts:  56%|#####5    | 5/9 [00:00<00:00, 68.24it/s]
[2024-08-12T06:01:00.095+0000] {logging_mixin.py:151} WARNING - Downloading artifacts:  67%|######6   | 6/9 [00:00<00:00, 68.44it/s]
[2024-08-12T06:01:00.103+0000] {logging_mixin.py:151} WARNING - Downloading artifacts:  78%|#######7  | 7/9 [00:00<00:00, 72.74it/s]
[2024-08-12T06:01:00.107+0000] {logging_mixin.py:151} WARNING - Downloading artifacts:  89%|########8 | 8/9 [00:00<00:00, 80.00it/s]
[2024-08-12T06:01:01.188+0000] {logging_mixin.py:151} WARNING - Downloading artifacts: 100%|##########| 9/9 [00:01<00:00,  7.62it/s]
[2024-08-12T06:01:01.188+0000] {logging_mixin.py:151} WARNING - Downloading artifacts: 100%|##########| 9/9 [00:01<00:00,  7.62it/s]
[2024-08-12T06:01:01.188+0000] {logging_mixin.py:151} WARNING - Downloading artifacts: 100%|##########| 9/9 [00:01<00:00,  7.62it/s]
[2024-08-12T06:01:01.397+0000] {logging_mixin.py:151} INFO - RandomForestClassifier version 1 loaded
[2024-08-12T06:01:34.768+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-08-12T06:01:34.770+0000] {process_utils.py:131} INFO - Sending 15 to group 7285. PIDs of all processes in the group: [7285]
[2024-08-12T06:01:34.771+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 7285
[2024-08-12T06:01:34.772+0000] {taskinstance.py:1630} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-08-12T06:01:34.828+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=7285, status='terminated', exitcode=0, started='06:00:58') (7285) terminated with exit code 0
